src/main/java/org/apache/sling/resourceresolver/impl/mapping/MapEntries.java

```diff
@@ -1393,6 +1437,27 @@ public Resource next() throws NoSuchElementException {
             next = null;
             return result;
         }
+
+        private void doPageStats() {
```

Comment:
The method name is a bit unspecific, maybe this?
```suggestion
        private void updatePageStats() {
```